packages/overlay/src/index.css

```diff
@@ -136,3 +136,32 @@ input:checked ~ .dot {
 .spotlight-debugger.resizing-row * {
   cursor: row-resize !important;
 }
+
+/* custom scrollbar */
+* {
+  scrollbar-width: thin;
+  scrollbar-color: theme('colors.primary.700') theme('colors.primary.900');
+}
+
+*::-webkit-scrollbar {
+  width: 6px;
+  height: 6px;
+}
+
+*::-webkit-scrollbar-track {
+  background: theme('colors.primary.900');
+  border-radius: 3px;
+}
```

Comment:
I think we should use the standardized `scrollbar-width` and `scrollbar-color` properties? -- https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar#adding_a_fallback_for_scrollbar_styles